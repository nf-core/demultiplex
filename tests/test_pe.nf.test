nextflow_pipeline {

    name "Test Workflow main.nf - Test PE"
    script "../main.nf"
    profile "test_pe"
    tag "bcl2fastq_pe"
    tag "pipeline"

    test("Bcl2Fastq_pe") {

        when {
            params {
                outdir        = "$outputDir"
            }
        }

        then {
            // stable_name: All files + folders in ${params.outdir}/ with a stable name
            def stable_name = getAllFilesFromDir(params.outdir, relative: true, includeDir: true, ignore: ['pipeline_info/*.{html,json,txt}'])
            // stable_path: All files in ${params.outdir}/ with stable content
            def stable_path = getAllFilesFromDir(params.outdir, ignoreFile: 'tests/.nftignore')
            assertAll(
                { assert workflow.success},
                { assert snapshot(
                    // Number of successful tasks
                    workflow.trace.succeeded().size(),
                    // pipeline versions.yml file for multiqc from which Nextflow version is removed because we tests pipelines on multiple Nextflow versions
                    removeNextflowVersion("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml"),
                    // All stable path name, with a relative path
                    stable_name,
                    // All files with stable contents
                    stable_path
                ).match() },
                { assert snapshot(UTILS.validateFastqPaths("$outputDir/samplesheet/atacseq_samplesheet.csv")).match("atacseq_samplesheet") },
                { assert snapshot(UTILS.validateFastqPaths("$outputDir/samplesheet/rnaseq_samplesheet.csv")).match("rnaseq_samplesheet") },
                { assert snapshot(UTILS.validateFastqPaths("$outputDir/samplesheet/taxprofiler_samplesheet.csv")).match("taxprofiler_samplesheet") }
            )
        }
    }
}
