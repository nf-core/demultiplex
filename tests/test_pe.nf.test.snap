{
    "rnaseq_samplesheet": {
        "content": [
            true
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-10-30T15:49:50.982685279"
    },
    "atacseq_samplesheet": {
        "content": [
            true
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-10-30T15:49:40.878739318"
    },
    "Bcl2Fastq_pe": {
        "content": [
            9,
            {
                "BCL2FASTQ": {
                    "bcl2fastq": "2.20.0.422"
                },
                "FALCO": {
                    "falco": "1.2.1"
                },
                "FASTP": {
                    "fastp": "0.23.4"
                },
                "MD5SUM": {
                    "md5sum": 8.3
                },
                "Workflow": {
                    "nf-core/demultiplex": "v1.5.3dev"
                }
            },
            [
                "PE_Sample",
                "PE_Sample.lane1_no_adapters.csv",
                "PE_Sample/InterOp",
                "PE_Sample/InterOp/IndexMetricsOut.bin",
                "PE_Sample/L001",
                "PE_Sample/L001/Reports",
                "PE_Sample/L001/Reports/html",
                "PE_Sample/L001/Reports/html/000000000-K9H97",
                "PE_Sample/L001/Reports/html/000000000-K9H97/all",
                "PE_Sample/L001/Reports/html/000000000-K9H97/all/all",
                "PE_Sample/L001/Reports/html/000000000-K9H97/all/all/all",
                "PE_Sample/L001/Reports/html/000000000-K9H97/all/all/all/lane.html",
                "PE_Sample/L001/Reports/html/000000000-K9H97/all/all/all/laneBarcode.html",
                "PE_Sample/L001/Reports/html/000000000-K9H97/all/all/unknown",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/Sample1",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/Sample1/all",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/Sample1/all/lane.html",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/Sample1/all/laneBarcode.html",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/Sample1/unknown",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/Sample1/unknown/lane.html",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/Sample1/unknown/laneBarcode.html",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/all",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/all/all",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/all/all/lane.html",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/all/all/laneBarcode.html",
                "PE_Sample/L001/Reports/html/000000000-K9H97/default/all/unknown",
                "PE_Sample/L001/Reports/html/Report.css",
                "PE_Sample/L001/Reports/html/index.html",
                "PE_Sample/L001/Reports/html/tree.html",
                "PE_Sample/L001/Sample1_S1_L001.fastp.html",
                "PE_Sample/L001/Sample1_S1_L001.fastp.json",
                "PE_Sample/L001/Sample1_S1_L001_1.fastp.fastq.gz",
                "PE_Sample/L001/Sample1_S1_L001_1.fastp.fastq.gz.md5",
                "PE_Sample/L001/Sample1_S1_L001_1.fastp.fastq.gz_fastqc_data.txt",
                "PE_Sample/L001/Sample1_S1_L001_1.fastp.fastq.gz_fastqc_report.html",
                "PE_Sample/L001/Sample1_S1_L001_1.fastp.fastq.gz_summary.txt",
                "PE_Sample/L001/Sample1_S1_L001_2.fastp.fastq.gz",
                "PE_Sample/L001/Sample1_S1_L001_2.fastp.fastq.gz.md5",
                "PE_Sample/L001/Sample1_S1_L001_2.fastp.fastq.gz_fastqc_data.txt",
                "PE_Sample/L001/Sample1_S1_L001_2.fastp.fastq.gz_fastqc_report.html",
                "PE_Sample/L001/Sample1_S1_L001_2.fastp.fastq.gz_summary.txt",
                "PE_Sample/L001/Sample1_S1_L001_R1_001.fastq.gz",
                "PE_Sample/L001/Sample1_S1_L001_R2_001.fastq.gz",
                "PE_Sample/L001/Stats",
                "PE_Sample/L001/Stats/AdapterTrimming.txt",
                "PE_Sample/L001/Stats/ConversionStats.xml",
                "PE_Sample/L001/Stats/DemultiplexingStats.xml",
                "PE_Sample/L001/Stats/DemuxSummaryF1L1.txt",
                "PE_Sample/L001/Stats/FastqSummaryF1L1.txt",
                "PE_Sample/L001/Stats/Stats.json",
                "multiqc",
                "multiqc/multiqc_data",
                "multiqc/multiqc_data/bcl2fastq-lane-stats-table.txt",
                "multiqc/multiqc_data/bcl2fastq_lane_counts.txt",
                "multiqc/multiqc_data/bcl2fastq_sample_counts.txt",
                "multiqc/multiqc_data/bcl2fastq_undetermined.txt",
                "multiqc/multiqc_data/fastp-insert-size-plot.txt",
                "multiqc/multiqc_data/fastp-seq-content-gc-plot_Read_1_After_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-content-gc-plot_Read_1_Before_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-content-gc-plot_Read_2_After_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-content-gc-plot_Read_2_Before_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-content-n-plot_Read_1_After_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-content-n-plot_Read_1_Before_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-content-n-plot_Read_2_After_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-content-n-plot_Read_2_Before_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-quality-plot_Read_1_After_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-quality-plot_Read_1_Before_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-quality-plot_Read_2_After_filtering.txt",
                "multiqc/multiqc_data/fastp-seq-quality-plot_Read_2_Before_filtering.txt",
                "multiqc/multiqc_data/fastp_filtered_reads_plot.txt",
                "multiqc/multiqc_data/fastqc-1-status-check-heatmap.txt",
                "multiqc/multiqc_data/fastqc-1_per_base_n_content_plot.txt",
                "multiqc/multiqc_data/fastqc-1_per_base_sequence_quality_plot.txt",
                "multiqc/multiqc_data/fastqc-1_per_sequence_gc_content_plot_Counts.txt",
                "multiqc/multiqc_data/fastqc-1_per_sequence_gc_content_plot_Percentages.txt",
                "multiqc/multiqc_data/fastqc-1_per_sequence_quality_scores_plot.txt",
                "multiqc/multiqc_data/fastqc-1_sequence_counts_plot.txt",
                "multiqc/multiqc_data/fastqc-1_sequence_duplication_levels_plot.txt",
                "multiqc/multiqc_data/fastqc-1_top_overrepresented_sequences_table.txt",
                "multiqc/multiqc_data/fastqc-2-status-check-heatmap.txt",
                "multiqc/multiqc_data/fastqc-2_per_base_n_content_plot.txt",
                "multiqc/multiqc_data/fastqc-2_per_base_sequence_quality_plot.txt",
                "multiqc/multiqc_data/fastqc-2_per_sequence_gc_content_plot_Counts.txt",
                "multiqc/multiqc_data/fastqc-2_per_sequence_gc_content_plot_Percentages.txt",
                "multiqc/multiqc_data/fastqc-2_per_sequence_quality_scores_plot.txt",
                "multiqc/multiqc_data/fastqc-2_sequence_counts_plot.txt",
                "multiqc/multiqc_data/fastqc-2_sequence_duplication_levels_plot.txt",
                "multiqc/multiqc_data/fastqc-2_top_overrepresented_sequences_table.txt",
                "multiqc/multiqc_data/multiqc.log",
                "multiqc/multiqc_data/multiqc_bcl2fastq_bylane.txt",
                "multiqc/multiqc_data/multiqc_bcl2fastq_bysample.txt",
                "multiqc/multiqc_data/multiqc_citations.txt",
                "multiqc/multiqc_data/multiqc_data.json",
                "multiqc/multiqc_data/multiqc_fastp.txt",
                "multiqc/multiqc_data/multiqc_fastqc.txt",
                "multiqc/multiqc_data/multiqc_fastqc_1.txt",
                "multiqc/multiqc_data/multiqc_general_stats.txt",
                "multiqc/multiqc_data/multiqc_software_versions.txt",
                "multiqc/multiqc_data/multiqc_sources.txt",
                "multiqc/multiqc_plots",
                "multiqc/multiqc_plots/pdf",
                "multiqc/multiqc_plots/pdf/bcl2fastq-lane-stats-table.pdf",
                "multiqc/multiqc_plots/pdf/bcl2fastq_lane_counts-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bcl2fastq_lane_counts-pct.pdf",
                "multiqc/multiqc_plots/pdf/bcl2fastq_sample_counts-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bcl2fastq_sample_counts-pct.pdf",
                "multiqc/multiqc_plots/pdf/bcl2fastq_undetermined-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bcl2fastq_undetermined-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastp-insert-size-plot.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-gc-plot_Read_1_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-gc-plot_Read_1_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-gc-plot_Read_2_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-gc-plot_Read_2_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-n-plot_Read_1_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-n-plot_Read_1_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-n-plot_Read_2_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-content-n-plot_Read_2_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-quality-plot_Read_1_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-quality-plot_Read_1_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-quality-plot_Read_2_After_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp-seq-quality-plot_Read_2_Before_filtering.pdf",
                "multiqc/multiqc_plots/pdf/fastp_filtered_reads_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/fastp_filtered_reads_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1-status-check-heatmap.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_base_n_content_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_base_sequence_quality_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_sequence_gc_content_plot_Counts.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_sequence_gc_content_plot_Percentages.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_per_sequence_quality_scores_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_sequence_counts_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_sequence_counts_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_sequence_duplication_levels_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-1_top_overrepresented_sequences_table.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2-status-check-heatmap.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2_per_base_n_content_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2_per_base_sequence_quality_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2_per_sequence_gc_content_plot_Counts.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2_per_sequence_gc_content_plot_Percentages.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2_per_sequence_quality_scores_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2_sequence_counts_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2_sequence_counts_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2_sequence_duplication_levels_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-2_top_overrepresented_sequences_table.pdf",
                "multiqc/multiqc_plots/pdf/general_stats_table.pdf",
                "multiqc/multiqc_plots/png",
                "multiqc/multiqc_plots/png/bcl2fastq-lane-stats-table.png",
                "multiqc/multiqc_plots/png/bcl2fastq_lane_counts-cnt.png",
                "multiqc/multiqc_plots/png/bcl2fastq_lane_counts-pct.png",
                "multiqc/multiqc_plots/png/bcl2fastq_sample_counts-cnt.png",
                "multiqc/multiqc_plots/png/bcl2fastq_sample_counts-pct.png",
                "multiqc/multiqc_plots/png/bcl2fastq_undetermined-cnt.png",
                "multiqc/multiqc_plots/png/bcl2fastq_undetermined-pct.png",
                "multiqc/multiqc_plots/png/fastp-insert-size-plot.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-gc-plot_Read_1_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-gc-plot_Read_1_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-gc-plot_Read_2_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-gc-plot_Read_2_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-n-plot_Read_1_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-n-plot_Read_1_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-n-plot_Read_2_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-content-n-plot_Read_2_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-quality-plot_Read_1_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-quality-plot_Read_1_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-quality-plot_Read_2_After_filtering.png",
                "multiqc/multiqc_plots/png/fastp-seq-quality-plot_Read_2_Before_filtering.png",
                "multiqc/multiqc_plots/png/fastp_filtered_reads_plot-cnt.png",
                "multiqc/multiqc_plots/png/fastp_filtered_reads_plot-pct.png",
                "multiqc/multiqc_plots/png/fastqc-1-status-check-heatmap.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_base_n_content_plot.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_base_sequence_quality_plot.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_sequence_gc_content_plot_Counts.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_sequence_gc_content_plot_Percentages.png",
                "multiqc/multiqc_plots/png/fastqc-1_per_sequence_quality_scores_plot.png",
                "multiqc/multiqc_plots/png/fastqc-1_sequence_counts_plot-cnt.png",
                "multiqc/multiqc_plots/png/fastqc-1_sequence_counts_plot-pct.png",
                "multiqc/multiqc_plots/png/fastqc-1_sequence_duplication_levels_plot.png",
                "multiqc/multiqc_plots/png/fastqc-1_top_overrepresented_sequences_table.png",
                "multiqc/multiqc_plots/png/fastqc-2-status-check-heatmap.png",
                "multiqc/multiqc_plots/png/fastqc-2_per_base_n_content_plot.png",
                "multiqc/multiqc_plots/png/fastqc-2_per_base_sequence_quality_plot.png",
                "multiqc/multiqc_plots/png/fastqc-2_per_sequence_gc_content_plot_Counts.png",
                "multiqc/multiqc_plots/png/fastqc-2_per_sequence_gc_content_plot_Percentages.png",
                "multiqc/multiqc_plots/png/fastqc-2_per_sequence_quality_scores_plot.png",
                "multiqc/multiqc_plots/png/fastqc-2_sequence_counts_plot-cnt.png",
                "multiqc/multiqc_plots/png/fastqc-2_sequence_counts_plot-pct.png",
                "multiqc/multiqc_plots/png/fastqc-2_sequence_duplication_levels_plot.png",
                "multiqc/multiqc_plots/png/fastqc-2_top_overrepresented_sequences_table.png",
                "multiqc/multiqc_plots/png/general_stats_table.png",
                "multiqc/multiqc_plots/svg",
                "multiqc/multiqc_plots/svg/bcl2fastq-lane-stats-table.svg",
                "multiqc/multiqc_plots/svg/bcl2fastq_lane_counts-cnt.svg",
                "multiqc/multiqc_plots/svg/bcl2fastq_lane_counts-pct.svg",
                "multiqc/multiqc_plots/svg/bcl2fastq_sample_counts-cnt.svg",
                "multiqc/multiqc_plots/svg/bcl2fastq_sample_counts-pct.svg",
                "multiqc/multiqc_plots/svg/bcl2fastq_undetermined-cnt.svg",
                "multiqc/multiqc_plots/svg/bcl2fastq_undetermined-pct.svg",
                "multiqc/multiqc_plots/svg/fastp-insert-size-plot.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-gc-plot_Read_1_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-gc-plot_Read_1_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-gc-plot_Read_2_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-gc-plot_Read_2_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-n-plot_Read_1_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-n-plot_Read_1_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-n-plot_Read_2_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-content-n-plot_Read_2_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-quality-plot_Read_1_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-quality-plot_Read_1_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-quality-plot_Read_2_After_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp-seq-quality-plot_Read_2_Before_filtering.svg",
                "multiqc/multiqc_plots/svg/fastp_filtered_reads_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/fastp_filtered_reads_plot-pct.svg",
                "multiqc/multiqc_plots/svg/fastqc-1-status-check-heatmap.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_base_n_content_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_base_sequence_quality_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_sequence_gc_content_plot_Counts.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_sequence_gc_content_plot_Percentages.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_per_sequence_quality_scores_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_sequence_counts_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_sequence_counts_plot-pct.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_sequence_duplication_levels_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-1_top_overrepresented_sequences_table.svg",
                "multiqc/multiqc_plots/svg/fastqc-2-status-check-heatmap.svg",
                "multiqc/multiqc_plots/svg/fastqc-2_per_base_n_content_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-2_per_base_sequence_quality_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-2_per_sequence_gc_content_plot_Counts.svg",
                "multiqc/multiqc_plots/svg/fastqc-2_per_sequence_gc_content_plot_Percentages.svg",
                "multiqc/multiqc_plots/svg/fastqc-2_per_sequence_quality_scores_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-2_sequence_counts_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/fastqc-2_sequence_counts_plot-pct.svg",
                "multiqc/multiqc_plots/svg/fastqc-2_sequence_duplication_levels_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc-2_top_overrepresented_sequences_table.svg",
                "multiqc/multiqc_plots/svg/general_stats_table.svg",
                "multiqc/multiqc_report.html",
                "pipeline_info",
                "pipeline_info/nf_core_pipeline_software_mqc_versions.yml",
                "samplesheet",
                "samplesheet/atacseq_samplesheet.csv",
                "samplesheet/rnaseq_samplesheet.csv",
                "samplesheet/taxprofiler_samplesheet.csv"
            ],
            [
                "PE_Sample.lane1_no_adapters.csv:md5,c31cca10b3a1aed8ec68daf779217d91",
                "IndexMetricsOut.bin:md5,9e688c58a5487b8eaf69c9e1005ad0bf",
                "Report.css:md5,eb7d3eb68fc1539f411404987246b59b",
                "index.html:md5,5747c407854ae2c358d0ec201ce622d8",
                "tree.html:md5,a1b9bf592973ca829ec69ddf888b7e34",
                "Sample1_S1_L001.fastp.json:md5,ad6e2f9b58a9d677f4eaa50353e6071d",
                "Sample1_S1_L001_1.fastp.fastq.gz:md5,64639ff33d4b07d62a9079f88cbbd373",
                "Sample1_S1_L001_1.fastp.fastq.gz.md5:md5,d9e351a8a996e158aec1be44440edec0",
                "Sample1_S1_L001_1.fastp.fastq.gz_fastqc_data.txt:md5,2de491ae04c8eb50d4a6a5759ade8322",
                "Sample1_S1_L001_1.fastp.fastq.gz_summary.txt:md5,917b542afe3e3f178b994317c955ce02",
                "Sample1_S1_L001_2.fastp.fastq.gz:md5,adbc8e47c8e7e902777a01ea3d73b306",
                "Sample1_S1_L001_2.fastp.fastq.gz.md5:md5,bb896815f6a2b678fe3451b80ed8c653",
                "Sample1_S1_L001_2.fastp.fastq.gz_fastqc_data.txt:md5,88cc69dd5e9eb06d6045c8d9e9a98af3",
                "Sample1_S1_L001_2.fastp.fastq.gz_summary.txt:md5,441197ccd90d2b018db631a66d6fbbb3",
                "Sample1_S1_L001_R1_001.fastq.gz:md5,0675fb6365322eaafb33c0f8e862b54b",
                "Sample1_S1_L001_R2_001.fastq.gz:md5,146eefcab1eaaef17ffb72672c021368",
                "AdapterTrimming.txt:md5,8009886a4c060bfec6d8e9346feb0988",
                "DemultiplexingStats.xml:md5,2047ff18f5b9107c084de06e9ff943ad",
                "FastqSummaryF1L1.txt:md5,0274549f9541b1ba7828ae6e9c5bcbbd",
                "bcl2fastq-lane-stats-table.txt:md5,0c16ec8dd4fdf53afcb6af57272ad92d",
                "bcl2fastq_lane_counts.txt:md5,d6054e1ec3c2e8d33c3fdee5b250dfbc",
                "bcl2fastq_sample_counts.txt:md5,93cb0cd1aee1b464f0e640e1c3d0c142",
                "fastp-insert-size-plot.txt:md5,16f66543e78b50ef96e1ef15d0f46087",
                "fastp-seq-content-gc-plot_Read_1_After_filtering.txt:md5,8105d4ce1d752e1f8a26e4291e8d9c6c",
                "fastp-seq-content-gc-plot_Read_1_Before_filtering.txt:md5,0ac68e41c4de8ca35a1cd6ee7c6cb3f3",
                "fastp-seq-content-gc-plot_Read_2_After_filtering.txt:md5,9d3b2d8218d60f475e1059fe77bc0792",
                "fastp-seq-content-gc-plot_Read_2_Before_filtering.txt:md5,08cf2f62762c6b2a17c53299d63bb482",
                "fastp-seq-content-n-plot_Read_1_After_filtering.txt:md5,1ec69a9fd0d46a39dfb59e377733e98b",
                "fastp-seq-content-n-plot_Read_1_Before_filtering.txt:md5,1ec69a9fd0d46a39dfb59e377733e98b",
                "fastp-seq-content-n-plot_Read_2_After_filtering.txt:md5,1ec69a9fd0d46a39dfb59e377733e98b",
                "fastp-seq-content-n-plot_Read_2_Before_filtering.txt:md5,1ec69a9fd0d46a39dfb59e377733e98b",
                "fastp-seq-quality-plot_Read_1_After_filtering.txt:md5,1701b40c23dca068210f5410788e3be1",
                "fastp-seq-quality-plot_Read_1_Before_filtering.txt:md5,35de9aebdc467b1bbc9fd1f40d3aece4",
                "fastp-seq-quality-plot_Read_2_After_filtering.txt:md5,feb0492fc4a011f49b0ae771af00fad2",
                "fastp-seq-quality-plot_Read_2_Before_filtering.txt:md5,ced1654945cd8c6a85dbe3353959e4b1",
                "fastp_filtered_reads_plot.txt:md5,136a1d755ba6e790227f80cfcb4c1d62",
                "fastqc-1_per_base_n_content_plot.txt:md5,2c2e1e7e982151328380d462bf3da96a",
                "fastqc-1_per_base_sequence_quality_plot.txt:md5,16c2db072409c63e9e72cc8b8efd7c1d",
                "fastqc-1_per_sequence_gc_content_plot_Counts.txt:md5,9e2aa8732545bf8544e03e8451d5d200",
                "fastqc-1_per_sequence_gc_content_plot_Percentages.txt:md5,2dc5206a46e9b07e971384413ce814c4",
                "fastqc-1_per_sequence_quality_scores_plot.txt:md5,17187938a14b3380211306d823bcb3f8",
                "fastqc-1_sequence_counts_plot.txt:md5,5ab45b6a9e87a73e919c9f27d686bf8d",
                "fastqc-1_sequence_duplication_levels_plot.txt:md5,47853398841fe7f3f8e2c930b8674247",
                "fastqc-1_top_overrepresented_sequences_table.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                "fastqc-2_per_base_n_content_plot.txt:md5,ecb4ae5f410a79977f13f53743e61d88",
                "fastqc-2_per_base_sequence_quality_plot.txt:md5,dbefa9916514fc5161c358579fbf6abd",
                "fastqc-2_per_sequence_gc_content_plot_Counts.txt:md5,f1efe8f47e924cadd8a9f64b63b53d4d",
                "fastqc-2_per_sequence_gc_content_plot_Percentages.txt:md5,9082529ef49895ae7cad91915745dd0d",
                "fastqc-2_per_sequence_quality_scores_plot.txt:md5,9e268b16fcd8a42e771dca575a70e592",
                "fastqc-2_sequence_counts_plot.txt:md5,266ea49ebc560e003e3dd1b9f4a21e0d",
                "fastqc-2_sequence_duplication_levels_plot.txt:md5,02c6dcf1a65b7b3a21d8deaf70c8f3ca",
                "fastqc-2_top_overrepresented_sequences_table.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                "multiqc_bcl2fastq_bysample.txt:md5,bc5b6bbdcd75ead5a552cef09fc9aaec",
                "multiqc_citations.txt:md5,d35df50e9903a96a2b3bce3c1fbc8ad2"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.1"
        },
        "timestamp": "2024-11-21T19:07:38.034440806"
    },
    "taxprofiler_samplesheet": {
        "content": [
            true
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-10-30T15:50:01.08084426"
    }
}